dali/sasha/saxref.cpp

```diff
@@ -716,7 +717,8 @@ class CNewXRefManager: public CNewXRefManagerBase
         if (plane)
         {
             storagePlane.set(plane);
-            isPlaneStriped = !storagePlane->hasProp("@hostGroup") && (storagePlane->getPropInt("@numDevices")>1);
+            numDevices = storagePlane->getPropInt("@numDevices", 1);
```

Comment:
need to be careful where used, e.g. on line 1326.. it is being used to indicate striping too, which will be wrong if `@hostGroup`.
I think you rename this to numStripedDevices - and set:
```
numStripedDevices = isPlaneStriped ? storagePlane->getPropInt("@numDevices", 1) : 1
```